views/partials/tips.hbs

```diff
@@ -1,4 +1,175 @@
+
 {{!< default }}
+
+<div id="password-advice" class="section-container">
+  <div class="section-wrapper">
+  <h3>Passwords</h3>
+  <h5>The Key to Account Security</h5>
+  <p>Your passwords protect all of the personal information that resides in your online accounts, from bank balances and credit card numbers to your home address and private messages. Passwords are so valuable that thousands of passwords are stolen every day and accounts are traded on the black market.  Take these six steps to protect yours.</p>
+  </div>
+</div>
+<div id="tips-container" class="section-container">
+    <div class="tips-wrapper">
+      <div class="tip-container">
+          <div class="tip">
+              <div class="svg-background">
+                <img src="img/Password_Tip_1@2x.png" alt="Account Security Tip #1">
+                <span class="tip-numeral medium">1</span>
+              </div>
+              <div class="tip-header">
+                  <h3>Use a different password<br>for every account</h3>
+              </div>
+              <p>You can't prevent a data breach, but you can limit your exposure by always using different passwords for different websites.</p>
+          </div><!--closes tip-->
+      </div><!--closes tip-container 1-->
+
+      <!-- tip # 2 -->
+
+      <div class="tip-container">
+          <div class="tip">
+              <div class="svg-background">
+                <img src="img/Password_Tip_2@2x.png" alt="Account Security Tip #2">
```

Comment:
I'll let @nhnt11 or @groovecoder make an official call, but should these `alt=""` tags say this ^^^, or something like:

```html
alt="Create strong passwords"
```
... where that text is the `div.tip-header` text below (minus HTML formatting tags).

Reply:
That sounds like a good tweak to change the `alt` content to match the `div.tip-header` content.